store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -105,6 +115,243 @@ public void testWriteAndReadFiles() throws IOException, InterruptedException {
     FileUtils.deleteDirectory(new File(path));
   }
 
+  @Test
+  public void testReadWithFilterOfTransactional() throws IOException, InterruptedException {
```

Comment:
All these code changes of this PR is for NonTransactional table.  I think test case for Transactional table is redundant.

Reply:
I think we should add test case for Transactional table , ensure the Transactional table  is available too